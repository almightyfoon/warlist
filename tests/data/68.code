c5521b464E4l3y3ydPfW2G2H3u383xhT___-0zdU4E1g0u9v3y3y4141dPfW4J35363xhT