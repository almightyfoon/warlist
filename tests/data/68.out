https://conflictchamber.com/#c5521b464Emr3y3ydPfWmm2H3u383xhT___-0zdU4E1gmv9v3y3y4141dPfW4J35363xhT

Retribution Army - 73 / 75 points

[Ossyan 1] Lord Arcanist Ossyan [+28]
 - Banshee [18]
 - Hypnos [17]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Houseguard Thane [4]
Lys Healer [3]
Dawnguard Invictors (max) [15]
 - Dawnguard Invictor Officer & Standard [4]
 - Soulless Escort (1) [1]
Houseguard Riflemen (min) [9]
Stormfall Archers [9]
Arcantrik Force Generator [17]

Retribution Army - 74 / 75 points
[Theme] Defenders of Ios

[Issyria 1] Issyria, Sibyl of Dawn [+29]
 - Banshee [18]
 - Chimera [8]
 - Phoenix [17]
 - Sylys Wyshnalyrr, the Seeker [0(4)]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Ghost Sniper [3]
Ghost Sniper [3]
Houseguard Thane [0(4)]
Lys Healer [3]
Heavy Rifle Team [4]
Houseguard Halberdiers (max) [13]
 - Houseguard Halberdier Officer & Standard [4]
Stormfall Archers [9]
Arcantrik Force Generator [17]
